doc/1/plugins/guides/events/core-hotelclerk-addSubscription/index.md

```diff
@@ -0,0 +1,29 @@
+---
+code: true
+type: page
+title: core:hotelClerk:addSubscription
+---
+
+# core:hotelClerk:addSubscription
+
+
+| Arguments  | Type              | Description                           |
+| ---------- | ----------------- | ------------------------------------- |
+| `diff`     | <pre>object</pre> | Contains information about subscription |
```

Comment:
We use `diff` internally because we use that information as a diff to synchronize with other cluster nodes, but here it makes little sense.

```suggestion
| `subscription`     | <pre>object</pre> | Contains information about the added subscription |
```